node_groups.tf

```diff
@@ -6,17 +6,16 @@ locals {
     launch_template_use_name_prefix = true
     launch_template_name            = var.default_group_launch_template_name
 
-    platform      = "bottlerocket"
-    ami_id        = coalesce(var.default_group_ami_id, data.aws_ami.eks_default_bottlerocket.id)
-    instance_type = var.default_group_instance_type
+    platform       = "bottlerocket"
+    ami_id         = coalesce(var.default_group_ami_id, data.aws_ami.eks_default_bottlerocket.id)
```

Comment:
@thepoppingone here we are already using bottlerocket for default group.

Reply:
somehow, if we use the submodule (eks_managed_nodes) without the cluster module, the submodule is still using Amazon Linux 2. I just tried to create a new node group (without a new cluster) the default remains to be not bottlerocket

Reply:
@niroz89 he means the default in the submodule.

Reply:
@thepoppingone default bottlerocket is set at the submodule too.